02.extensions/zoocart/01.basics/06.faq/docs.md

```diff
@@ -16,3 +16,4 @@ taxonomy:
 
 * **Why shipping rates is not assigning automatically?** <br /> Ensure that all conditions of shipping rate are true for current order.
 
+* **How can a [Subscribtion](/extensions/zoocart/basics/integration#subscriptions) be renewed from the frontend?** <br /> On the [Subscriptions page](/extensions/zoocart/basics/integration#menu-assignment), the "Renew" button is available, after a Subscription is ended (if this product still exists). After click the Subscription product will be added to the Cart. Then the user follows the standard checkout process.
```

Comment:
After clicking on the Renew button the product will be added to the Cart and the user redirected to the Checkout.